src/lib/github/addIssuesToRepo.ts

```diff
@@ -19,6 +20,7 @@ export default async function addIssuesToRepo (context: Context, newIssues: Issu
       body,
       title,
       labels: ["GH-TODO-BOT"],
+      assignees: author,
```

Comment:
I remember as if `assignees` is of array type. Can you confirm that

Reply:
true

Reply:
You shouldn't resolve issues manually until they have been addressed.